tests/utils/command_line_params.h

```diff
@@ -7,9 +7,18 @@
 // for compatibility while porting - remove later
 extern void usage(char **);
 
+// Put this is quda_constants.h?
```

Comment:
The answer to this question depends if these are universal constants or limited only to the tests?

Reply:
Ping on this comment